x/pairing/keeper/epoch_cu.go

```diff
@@ -36,7 +36,7 @@ func (k Keeper) IsUniqueEpochSessionExists(ctx sdk.Context, epoch uint64, provid
 // RemoveAllUniqueEpochSession removes all the UniqueEpochSession objects from the store for a specific epoch
 func (k Keeper) RemoveAllUniqueEpochSession(ctx sdk.Context, epoch uint64) {
 	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.UniqueEpochSessionKeyPrefix())
-	iterator := sdk.KVStorePrefixIterator(store, types.EncodeBlock(epoch))
+	iterator := sdk.KVStorePrefixIterator(store, utils.Serialize(epoch))
```

Comment:
need to make sure we dont change the serialization method

Reply:
yeah, I fixed here https://github.com/lavanet/lava/commit/ad7d7f8cc981dc891e09c8ddef560e7ea24b1d37 (reverted the Serialize method to the previous implementation)